libminifi/test/opencv-tests/CMakeLists.txt

```diff
@@ -24,14 +24,14 @@ SET(OPENCV_TEST_COUNT 0)
 FOREACH(testfile ${OPENCV_TESTS})
```

Comment:
I think opencv tests should be moved to the opencv extension as opencv tests hardly make sense without the processor.